addons/website_livechat/static/tests/tours/website_livechat_chatbot_flow.js

```diff
@@ -1,200 +1,260 @@
+            },
+            {
+                trigger: messagesContain(
+                    "Great, do you want to leave any feedback for us to improve?"
+                ),
+                // should ask for feedback now
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit Yes, actually, I'm glad you asked!",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit I think it's outrageous that you ask for all my personal information!",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit I will be sure to take this to your manager!",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "press Enter",
+            },
+            {
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run: "edit I want to say...",
+            },
+            {
+                // Simulate that the user is typing, so the chatbot shouldn't go to the next step
+                trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+                run() {
+                    let counter = window.debounceAnswerCount;
+                    const target = new TourHelpers(this.anchor);
+                    const delay =
+                        odoo.__WOWL_DEBUG__.root.env.services["mail.store"].Chatbot
+                            .MULTILINE_STEP_DEBOUNCE_DELAY_TOUR;
+                    setTimeout(() => {
+                        target.edit("Never mind!");
+                        setTimeout(() => {
+                            if (window.debounceAnswerCount <= counter) {
+                                console.error(
+                                    "Chatbot should stay in multi line step when user is typing."
+                                );
+                            }
+                            counter = window.debounceAnswerCount;
+                        }, delay);
+                    }, 100);
+                    setTimeout(() => {
+                        target.edit("Never mind!!!!");
+                        setTimeout(() => {
+                            if (window.debounceAnswerCount <= counter) {
+                                console.error(
+                                    "Chatbot should stay in multi line step if user isn't done typing."
+                                );
+                            }
+                        }, delay * 2);
+                    }, 700);
+                },
```

Comment:
100 -> "Never mind!"
600 -> check count has increased (chatbot is awaiting)
700 -> "Never mind!!!!"
1700 -> check count has increased (chatbot is awaiting)

The last one doesn't make sense with the `tmpAnswer`, but I suggest to scrap it altogether.
The tour should stay simple as this is heavily prone to race condition. Properly await then await, no concurrent `setTimeout()` with each having inner and non-awaited `setTimeout()`.

With my suggestions to just check for seconds of consecutive empty input, then just test that chatbot is not consuming the step when input is non-empty (counter has increased). And that's it :)